docs/plans/06-chordpro-parser-exporter-tests.md

```
@@ -0,0 +1,62 @@
+# Test Plan: ChordPro Parser & Exporter
+
+## Scope
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)